components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/SessionDataStore.java

```diff
@@ -458,6 +458,8 @@ private void deleteSTOREOperationsTask(Timestamp timestamp) {
             log.error(e.getMessage(), e);
             return;
         }
+        long cleanupLimit = TimeUnit.MINUTES.toNanos(FrameworkUtils.getCurrentStandardNano() - IdentityUtil
```

Comment:
Do the same fix similar to

Should be corrected as,

`long cleanupLimitNano = FrameworkUtils.getCurrentStandardNano() - TimeUnit.MINUTES.toNanos(IdentityUtil.getCleanUpTimeout()));`